### PR TITLE
Feature/frontend/drone fonctionnal tests

### PR DESCRIPTION
**Pull request by PabloPie**

### CODE REVIEW

**Comment by iamdey** [comment]
may be move the file `src/backend/Dockerfile.caliopen-node` into src/frontend ?

File: src/frontend/web_application/test/functional/protractor.conf.js [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
-    //   marionette: false,
-    //   'moz:firefoxOptions': {
-    //     // can't get this working :(
-    //     args: ['--UILocale en', '--safe-mode'],
-    //     prefs: {
-    //       intl: { accept_languages: 'en-US' },
-    //     },
-    //   },
-    // },
+    {
```
Comment (iamdey) [inline src/frontend/web_application/test/functional/protractor.conf.js]: can you rollback the changes for dev env : chrome only

File: src/frontend/web_application/test/functional/protractor.conf.js [hunk 2]
```diff
@@ -58,52 +57,54 @@ const cfg = {
...
 
   cfg.sauceUser = process.env.SAUCE_USERNAME;
   cfg.sauceKey = process.env.SAUCE_ACCESS_KEY;
+  cfg.sauceSeleniumAddress = process.env.SAUCE_ADDRESS + ':' + process.env.SAUCE_PORT + '/wd/hub';
+  cfg.sauceSeleniumUseHttp = true;
+  const branch = process.env.DRONE_BRANCH;
+
   cfg.multiCapabilities = [
     {
-      browserName: 'firefox',
```
Comment (iamdey) [inline src/frontend/web_application/test/functional/protractor.conf.js]: for saucelabs env : `firefox only` (saucelabs only provides firefox 45 for linux)

File: src/backend/Dockerfile.caliopen-node [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Image to do fonctionnal Selenium tests locally, with Chrome and Firefox
```
[the remark this answers is not in the document]
Comment (PabloPie) [reply src/backend/Dockerfile.caliopen-node]: Yeah, I'll do a quick doc on it. I think there is a better way to do Selenium tests locally though.

**Pull request merged**